lte/protos/subscriberdb.proto

```diff
@@ -213,14 +218,50 @@ message SubscribersDigest {
   string md5_base64_digest = 1;
 }
 
+message QueryFlatDigestRequest {
+  // digest is the deterministic digest of the full set of subscriber objects
+  // stored on the client side.
+  SubscribersDigest digest = 1;
+}
+
+message QueryFlatDigestResponse {
+  // no_updates is true if client's existing subscribers match those on the server.
+  bool no_updates = 1;
+  // digest is the deterministic flat digest of the full set of subscriber objects
+  // defined by the server side.
+  SubscribersDigest digest = 2;
```

Comment:
No need to return the digest here -- this endpoint just says whether the subs are all the same